pages/resources/family-center.mdx

```diff
@@ -0,0 +1,245 @@
+| totals              | object                                                                    | Object keyed by [display types](#display-type), with their values          |
+
+###### Action Structure
+
+| Field           | Type         | Description                                                                                                         |
+| --------------- | ------------ | ------------------------------------------------------------------------------------------------------------------- |
+| event_id        | snowflake    | The ID of the event action                                                                                          |
+| user_id         | snowflake    | The ID of the linked user                                                                                           |
+| entity_id       | snowflake    | The ID of the entity the action relates to (user, guild, or group DM) based off the [display type](#display-type)   |
+| display_type    | integer      | The [display type](#display-type) of the action, detailing what this action involved                                |
+
+###### Display Type
+
+The Display Type represents what kind of action the linked user engaged in.
+
+| Value     | Description                                                         |
+| --------- | ------------------------------------------------------------------- |
+| 1         | Users added within the last 7 days                                  |
+| 2         | Server joined within the last 7 days                                |
+| 3         | Users messaged within the last 7 days                               |
+| 4         | Servers the linked user has sent messages in within the last 7 days |
+| 5         | Users called within the last 7 days                                 |
+
+###### Example Teen Audit Log
+
+```json
+{
+  "teen_user_id": "801318363472330772",
+  "range_start_id": "1328607677722394624",
+  "actions": [
+    {
+      "event_id": "1331144363278860318",
+      "user_id": "801318363472330772",
+      "entity_id": "246877849162743818",
+      "display_type": 3
+    },
+  ],
+  "users": [
+    {
+      "id": "246877849162743818",
+      "username": "jay_taelien",
+      "global_name": "Jay",
+      "avatar": "91b7bc37e924f78625f7ea582fdbac5d",
+      "avatar_decoration_data": {
+        "asset": "a_aa2e1c2b3cf05b24f6ec7b8b4141f5fc",
+        "sku_id": "1144056631374647458",
+        "expires_at": null
+      },
+      "discriminator": "0",
+      "public_flags": 16512,
+      "primary_guild": {
+        "identity_guild_id": "1241115476021481582",
+        "identity_enabled": true,
+        "tag": "BUG",
+        "badge": "6cdb13f9bdba1234e711a7c24dce1f51"
+      },
+      "clan": {
+        "identity_guild_id": "1241115476021481582",
+        "identity_enabled": true,
+        "tag": "BUG",
+        "badge": "6cdb13f9bdba1234e711a7c24dce1f51"
+      }
```

Comment:
```suggestion
      "primary_guild": null
```
in the interest of not having to update 3 million examples every time they update the user object, dead fields are omitted and optional structs are usually null